build_tools/docker/context/install_bazel.sh

```diff
@@ -17,12 +17,15 @@ BAZEL_VERSION="$(cat .bazelversion)"
 # We could do the whole apt install dance, but this technique works across a
 # range of platforms, allowing us to use a single script. See
 # https://bazel.build/install/ubuntu#binary-installer
+machine=$(uname -m)
+if [ "$machine" == "aarch64" ]; then
+  machine="arm64"
+fi
```

Comment:
nit: We usually use uppercase for bash variables, and use `${...}` variable expansion (https://google.github.io/styleguide/shellguide.html#variable-expansion).

Also maybe `ARCH` is a clearer name than `machine`?

```suggestion
ARCH="$(uname -m)"
if [ "${ARCH}" == "aarch64" ]; then
  ARCH="{arm64}"
fi
```

Reply:
WIll fix. (The `-m` is for machine, but I agree ARCH is a better name)